examples/particles/fireworks/fireworks.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Particle effect example - fireworks
+brief: This example shows a foreworks effect made with particles.
+scripts: fireworks.script
```

Comment:
This should be a comma separated list of scripts and modules to show on the example page. Please add fireworks.lua too and remove the `modules`.